packages/react-reconciler/src/ReactFiberDevToolsHook.new.js

```diff
@@ -171,3 +178,44 @@ export function onCommitUnmount(fiber: Fiber) {
     }
   }
 }
+
+let isStrictMode = false;
+
+export function setIsStrictModeForDevtools(newIsStrictMode: boolean) {
+  isStrictMode = newIsStrictMode;
+  if (consoleManagedByDevToolsDuringStrictMode) {
+    // We're in a test because Scheduler.unstable_yieldValue only exists
+    // in SchedulerMock. To reduce the noise in strict mode tests,
+    // suppress warnings and disable scheduler yielding during the double render
+    if (typeof unstable_yieldValue === 'function') {
```

Comment:
nit: The comment block ("We're in a test...") should maybe be moved inside of the `if` conditional, since that's when it's accurate.